Drawer/Drawer.js

```diff
@@ -47,13 +47,15 @@ const DrawerBase = kind({
 		// and the .closeButton CSS in the LESS file all relate to this functionality.
 		// onClose: PropTypes.func,
 		onHide: PropTypes.func,
-		open: PropTypes.bool
+		open: PropTypes.bool,
+		role: PropTypes.string
 	},
 
 	defaultProps: {
 		noAnimation: false,
 		open: false,
-		orientation: 'vertical'
+		orientation: 'vertical',
+		role: 'alert'
```

Comment:
I think we'd better remove this prop default props and just define this in jsx and open a way to override this.